homeassistant/components/launch_library/sensor.py

```diff
@@ -147,7 +163,16 @@ async def async_setup_entry(
             description=description,
             name=name if description.key == "next_launch" else None,
         )
-        for description in SENSOR_DESCRIPTIONS
+        for description in NEXT_LAUNCH_SENSOR_DESCRIPTIONS
+    )
+
+    async_add_entities(
+        StarshipSensor(
+            coordinator=coordinator,
+            entry_id=entry.entry_id,
+            description=description,
+        )
+        for description in STARSHIP_SENSOR_DESCRIPTIONS
     )
```

Comment:
Can this be done in a better way? mypy flags it, as a list can only contain one type. Can I pass a tuple instead?